day00/ex12/ex12.md

```diff
@@ -0,0 +1,27 @@
+# Exercise 12 - Worldwide
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex12              |
+|   Files to turn in :    |  worldwide.py      |
+|   Forbidden function :  |  None              |
+|   Remarks :             |  n/a               |
+
+## Objective
+
+Give the top 5 most played genres among games that have a number of distinct languages greater or equal to 3. 
+
+## Instructions
+
+You must write a query that will help you filter your games according to the number of languages they have, and then filter out the ones that have strictly less than 3 languages. Then you need to select the top 5 genres where those games appear.
```

Comment:
You must write a query ***which*** will help you **to** filter the games ...
At some point maybe you could ask to Amric to verify the english or any fluent english speaker

Reply:
remark will be fixed in another PR